zubhub_frontend/zubhub/src/components/upload_file/video.jsx

```diff
@@ -0,0 +1,248 @@
+                    <div
+                      className={
+                        classes.videoInputDialogControlButtonUseTextDescStyle
+                      }
+                    >
+                      {t('createProject.inputs.video.dialogURLToggle')}
+                    </div>
+                    <InsertLinkIcon
+                      className={
+                        classes.videoInputDialogControlButtonUseIconDescStyle
+                      }
+                    />
+                  </CustomButton>
+                  <CustomButton
+                    className={classes.videoInputDialogControlButtonStyle}
+                    primaryButtonStyle={select_video_file}
+                    secondaryButtonStyle={!select_video_file}
+                    size="medium"
+                    onClick={_ =>
+                      handleSetState(
+                        handleSelectVideoFileChecked(
+                          refs.video_file_el.current,
+                        ),
+                      )
+                    }
+                  >
+                    <div
+                      className={
+                        classes.videoInputDialogControlButtonUseTextDescStyle
+                      }
+                    >
+                      {t('createProject.inputs.video.dialogFileToggle')}
+                    </div>
+                    <MovieIcon
+                      className={
+                        classes.videoInputDialogControlButtonUseIconDescStyle
+                      }
+                    />
+                  </CustomButton>
+                </div>
+                <Grid container spacing={1} alignItems="flex-end">
+                  <Grid
+                    item
+                    xs={12}
+                    sm={12}
+                    className={clsx(
+                      common_classes.marginTop1em,
+                      classes.videoInputDialogBodyGridStyle,
+                    )}
+                  >
+                    {!select_video_file ? (
+                      <FormControl
+                        className={clsx(
+                          classes.margin,
+                          classes.textField,
+                          classes.videoInputDialogURLFormControlStyle,
+                        )}
+                        variant="outlined"
+                        size="small"
+                      >
+                        <InputLabel
+                          className={classes.customLabelStyle}
+                          htmlFor="url-input"
+                        >
+                          {t('createProject.inputs.video.dialogURLFieldLabel')}
+                        </InputLabel>
+                        <OutlinedInput
+                          ref={refs.video_el}
+                          className={classes.customInputStyle}
+                          type="text"
+                          name="url-input"
+                          labelWidth={calculateLabelWidth(
```

Comment:
Please check if you can achieve this with css instead of setting width through JS